docs/podman-create.1.md

```diff
@@ -470,6 +470,8 @@ Current supported mount TYPES are bind, and tmpfs.
 
        type=bind,source=/path/on/host,destination=/path/in/container
 
+       type=bind,src=/path/on/host,dst=/path/in/container,relabel=shared
+
```

Comment:
Just double checking.  `src` and `dst` are both valid?  The other examples have `source` and `destination`

Reply:
It is supported and documented in the man page